as2_utilities/as2_external_object_to_tf/tests/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# GTest
```

Comment:
Use documentation CMakeLists.txt for tests

Reply:
Documentation CMakeList do not work.

Reply:
So fix it and use it

Reply:
We have to document how we define interfaces in the same package.

Reply:
Done